Release v2.1/maatregel make or buy (#604)
MinBZK/Algoritmekader: docs/voldoen-aan-wetten-en-regels/maatregelen/2-owp-38-maak-of-koopbeslissing.md

---

> title: Bepaal of een algoritme moet worden gemaakt of ingekocht 

Reviewer: ```suggestion
title: Bepaal of een algoritme moet worden ontwikkeld of ingekocht
```

> id: urn:nl:ak:mtr:pba-03

Reviewer: ```suggestion
id: urn:nl:ak:mtr:owp-38
```

toelichting: Als in de ontwerpfase duidelijk is geworden welke doelstellingen en behoeften gerealiseerd moeten worden, dan moet de vraag worden beantwoord of het beoogde algoritme moet worden gemaakt of ingekocht.
vereiste:
- awb-01-zorgvuldigheidsbeginsel
levenscyclus:

> - ontwerpfase

Reviewer: ```suggestion
- ontwerp
```

onderwerp:
- publieke-inkoop
rollen:
- projectleider
sources:
  
hide:
- navigation
- toc
---

<!-- tags -->

## Maatregel

> Bepaal of een algoritme moet worden gemaakt of ingekocht  

Reviewer: ```suggestion
Bepaal of een algoritme moet worden ontwikkeld of ingekocht
```

## Toelichting

> Als in de ontwerpfase duidelijk is geworden welke doelstellingen en behoeften gerealiseerd moeten worden, dan moet de vraag worden beantwoord of het beoogde algoritme moet worden gemaakt of ingekocht.

Reviewer: ```suggestion
Als in de [probleemanalyse fase](../../levenscyclus/probleemanalyse.md) duidelijk is geworden welke doelstellingen en behoeften gerealiseerd moeten worden, dan moet in de ontwerpfase worden bepaald of het beoogde algoritme zelf moet worden ontwikkeld of moet worden ingekocht.
```

> Er zijn grofweg vier oplossingenrichtingen. 

Reviewer: ```suggestion
Er zijn grofweg vier oplossingsrichtingen. 
```

> - Een kant-en-klare oplossingen inkopen (off the shelf) bij een aanbieder.

Reviewer: ```suggestion
- Een kant-en-klare oplossing inkopen ("off the shelf") bij een aanbieder.
```

- Een kant-en-klare oplossing inkopen bij een aanbieder waar nog enig maatwerk bij nodig is.
- Een nieuwe oplossing laten ontwikkelen door een aanbieder.
- Een nieuwe oplossing binnen de eigen organisatie ontwikkelen.

Het is van belang dat een analyse wordt gedaan welke oplossingsrichting het meest wenselijk is. PIANOo heeft een bruikbaar (algemeen) afwegingskader en stappenplan opgesteld om deze analyse grondig uit te voeren. Denk hierbij aan maken van:

> - Een kwantitatieve kostenanalyse
> - Een kwalitatieve productiecriteria
> - Een kwalitatieve uitbestedingscriteria

Reviewer: ```suggestion
- Een kwantitatieve kostenanalyse
- Kwalitatieve productiecriteria
- Kwalitatieve uitbestedingscriteria
```

> Met een grondige analyse kan worden voorkomen dat bijvoorbeeld de ontwikkeling van een algoritme onbeheersbaar wordt omdat er een betere inschatting is gemaakt van de bijbehorende kosten of dat een algoritme zelfstandig wordt ontwikkeld wat al bestaat.

Reviewer: ```suggestion
Met een grondige analyse kan worden voorkomen dat bijvoorbeeld een algoritme dat al bestaat zelfstandig wordt ontwikkeld of dat de ontwikkeling van een algoritme onbeheersbaar wordt. Er is bij zo'n analyse een betere inschatting gemaakt van de voor en nadelen en bijbehorende kosten.
```

## Risico

> Al ontwikkelde algoritmes die aansluiten bij de behoeftenstelling en de bijbehordende vereisten worden onnodig opnieuw ontwikkeld of het zelfstandig ontwikkelen is verkeerd ingeschat waardoor de ontwikkeling te duur of onbeheersbaar wordt.

Reviewer: ```suggestion
Al ontwikkelde algoritmes die aansluiten bij de behoeftestelling worden onnodig opnieuw ontwikkeld of het zelfstandig ontwikkelen is verkeerd ingeschat waardoor de ontwikkeling te duur of onbeheersbaar wordt.
```

 
## Bijbehorende vereiste(n) { data-search-exclude }
??? expander "Bekijk alle vereisten"
    <!-- list_vereisten_on_maatregelen_page -->

> ## Bronnen
> 
> - [Maak of koopbeslissing, PIANOo](https://www.pianoo.nl/nl/inkoopproces/fase-1-voorbereiden-inkoopopdracht/aanbestedingsplicht/maak-of-koopbeslissing)
> - [Afwegingskader en stappenplan maak/koopbeslissing](https://www.pianoo.nl/nl/inkoopproces/economische-afwegingen-bij-aanbesteding/maak-of-koopbeslissing/afwegingskader-stappenplan-maakkoopbeslissing)

Reviewer: ```suggestion
## Bronnen

- [Maak of koopbeslissing (PIANOo)](https://www.pianoo.nl/nl/inkoopproces/fase-1-voorbereiden-inkoopopdracht/aanbestedingsplicht/maak-of-koopbeslissing)
- [Afwegingskader en stappenplan maak/koopbeslissing (PIANOo)](https://www.pianoo.nl/nl/inkoopproces/economische-afwegingen-bij-aanbesteding/maak-of-koopbeslissing/afwegingskader-stappenplan-maakkoopbeslissing)
```

- Een kwantitatieve kostenanalyse
- Kwalitatieve productiecriteria
- Kwalitatieve uitbestedingscriteria

Met een grondige analyse kan worden voorkomen dat bijvoorbeeld een algoritme dat al bestaat zelfstandig wordt ontwikkeld of dat de ontwikkeling van een algoritme onbeheersbaar wordt. Er is bij zo'n analyse een betere inschatting gemaakt van de voor en nadelen en bijbehorende kosten.

## Risico
Al ontwikkelde algoritmes die aansluiten bij de behoeftestelling worden onnodig opnieuw ontwikkeld of het zelfstandig ontwikkelen is verkeerd ingeschat waardoor de ontwikkeling te duur of onbeheersbaar wordt.
 
## Bijbehorende vereiste(n) { data-search-exclude }
??? expander "Bekijk alle vereisten"
    <!-- list_vereisten_on_maatregelen_page -->

## Bronnen

- [Maak of koopbeslissing (PIANOo)](https://www.pianoo.nl/nl/inkoopproces/fase-1-voorbereiden-inkoopopdracht/aanbestedingsplicht/maak-of-koopbeslissing)
- [Afwegingskader en stappenplan maak/koopbeslissing (PIANOo)](https://www.pianoo.nl/nl/inkoopproces/economische-afwegingen-bij-aanbesteding/maak-of-koopbeslissing/afwegingskader-stappenplan-maakkoopbeslissing)

> ## Voorbeeld
> 
> Heb jij een goed voorbeeld? Laat het ons weten!

Reviewer: ```suggestion
## Voorbeelden

Heb je een voorbeeld of best practice, laat het ons weten via [algoritmes@minbzk.nl](mailto:algoritmes@minbzk.nl)
```
